internal/lifecycle/lifecycle.go

```diff
@@ -46,7 +51,24 @@ func (l Lifecycle) Execute(commandsDir string) error {
 // NewLifecycle tries to parse the different options provided for build and deploy command configuration. If none of
 // those options are set up, it falls back to reasonable defaults based on whether the sample is java-based
 // (has a pom.xml) that doesn't have a Dockerfile or isn't.
-func NewLifecycle(sampleDir, serviceName, gcrURL string) (Lifecycle, error) {
+func NewLifecycle(sampleDir, serviceName, gcrURL, runRegion string, cloudBuildConfSubs map[string]string) (Lifecycle, error) {
+	// First try Cloud Build Config file
+	cloudBuildConfigPath := fmt.Sprintf("%s/cloudbuild.yaml", sampleDir)
+
+	_, err := os.Stat(cloudBuildConfigPath)
+	cloudBuildConfigE := err == nil
+
+	if cloudBuildConfigE {
```

Comment:
Any reason not to just inline this? 

```suggestion
	if err == nil {
```

Reply:
Resolved, thank you.